docs/source/contribute/development/developer_guide/release_instructions.rst

```diff
@@ -0,0 +1,182 @@
+.. _release:
+
+Release Management
+=====================================
+
+This section provides detailed instructions used to release a new version of ``skbase``.
+
+This task is carried out by core developers with write access to the ``skbase``
+repository and designated as release managers by the
+:ref:`Community Council <cc-members>`.
+
+Summary of release process
+--------------------------
+
+The release process includes, in sequence:
+
+* :ref:`prepare for an upcoming release <cycle_process>`
+* :ref:`get the release ready <release_version_prep>`
+* :ref:`release on PyPi <pypi_release>`
+* :ref:`release on conda <conda_release>`
+* troubeleshooting and accident remediation, if applicable (see troubeleshooting
+  tips in each of the prior sections)
+
+Details follow below.
+
+.. _cycle_process:
+
+Release preparation cycle
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``skbase`` aims for a release every month, typically coinciding with the start of
```

Comment:
well, let's just say I'll be happy if we can keep this promise :-)

Reply:
That's my intention. The hope is to get this release done, then take a few days to plan a reasonable slate of work to accopmlish by start of February and release v0.4.0.